try:
    result = 4/0
    print(result)
except Exception as e:
    print(e)

print("Program End")
